tests/python/unittest/test_init.py

```diff
@@ -44,8 +44,21 @@ def test_aux_init():
     assert (mod.get_params()[1]['bn_moving_var'].asnumpy() == 1).all()
     assert (mod.get_params()[1]['bn_moving_mean'].asnumpy() == 0).all()
 
+def test_rsp_const_init():
+    shape = (10, 100)
```

Comment:
nit: shape=`(10,10)` should be sufficient for unit test to verify correctness

Reply:
Can we also add test `initializer.Zero` and `initializer.One` to make it more complete?